### PR TITLE
Init Lana for Bacom

### PR DESCRIPTION
**Pull request by chrischrischris**
* Note that the lana file is not loaded until `window.lana.log` has been called, so there should be no changes to performance.

**Test URLs:**
- Before: https://main--bacom--adobecom.hlx.page/?martech=off
- After: https://c3-initlana--bacom--adobecom.hlx.page/?martech=off

### CODE REVIEW

**Comment by meganthecoder** [comment]
@chrischrischris Does this need to be tested? All bacom PRs not deemed "trivial" need to be verified by Dennis

**Review by meganthecoder — APPROVED** [review]

**Comment by chrischrischris** [comment]
@meganthecoder  Doesn't hurt to test it.

**Comment by meganthecoder** [comment]
> @meganthecoder Doesn't hurt to test it.

Sounds good. I've added him as the assignee. Once you have the approvals you need, please ping him to let him know it's ready to test. We usually do this in the ticket but since you don't have one, slack could be a good option. He will probably also need testing steps unless he's already familiar

**Review by JasonHowellSlavin — APPROVED** [review]

**Pull request merged**
